src/docs/development/platform-integration/platform-views.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Hosting native Android and iOS views in your Flutter app with Platform Views
+short-title: Platform-views
+description: Learn how to host native Android and iOS views in your Flutter app with Platform Views.
+---
+
+As of the Flutter 1.22 release, you can now host
+Android and iOS views from your Flutter mobile app.
+This allows you, for example, to use the native
+Google Maps from the Android and iOS SDKs
+directly inside your Flutter app, by using Platform Views.
+The [`google_maps_flutter`][] and [`webview_flutter`][]
+plugins are both updated (and are in `developer_preview`)
+with the new functionality.
+
+[`google_maps_flutter`]: {{site.pub}}/packages/google_maps_flutter
+[`webview_flutter`]: {{site.pub}}/packages/webview_flutter
+
+This page discusses how to host your own native components
+within a Flutter app.
+
+## Hybrid composition on Android
+
+Hybrid composition on Android is enabled by default,
+and only requires API level 19 (instead of API level 20,
+as the previous version required).
```

Comment:
```suggestion
Flutter supports two modes: Virtual displays and Hybrid composition.

In 1.22, Hybrid composition is enabled by default, 
and only requires API level 19 (instead of API level 20,
as the previous version required).
```

Reply:
Would be useful to have some explanation of the tradeoff between the two